docs/sources/collect/choose-component.md

```diff
@@ -17,10 +17,9 @@ The components you select and configure depend on the telemetry signals you want
 ## Metrics for infrastructure
 
 Use `prometheus.*` components to collect infrastructure metrics.
-This will give you the best experience with [Grafana Infrastructure Observability][].
+This gives you the best experience with [Grafana Infrastructure Observability][].
```

Comment:
empty brackets are okay here?

Reply:
The link is defined on line 27, so it should be OK